src/services/circles.d.ts

```diff
@@ -24,9 +24,9 @@ interface MemberPairs {
     id: string;
     type: MemberType;
 }
-declare type CircleEditType = 'displayName' | 'description' | 'settings' | 'config';
+declare type CircleEditType = 'name' | 'description' | 'settings' | 'config';
 export declare enum CircleEdit {
-    DisplayName = "displayName",
+    Name = "displayName",
```

Comment:
```suggestion
    Name = "name",
```

Reply:
That would require the circles route to change first. But as @daita already planned to do it, let's do it now ;)

I just opened https://github.com/nextcloud/circles/pull/619